tokio/src/io/interest.rs

```diff
@@ -167,6 +167,37 @@ impl Interest {
         Self(self.0 | other.0)
     }
 
+    /// Remove `Interest` from `self`.
+    ///
+    /// Returns `None` if the set would be empty after removing `Interest`.
```

Comment:
I would like to mention that any interests in `other` but not in `self` are ignored.